src/contexts/index.ts

```diff
@@ -1,20 +1,24 @@
 import { contextFactory } from 'config/context';
+import { Page } from 'types/types';
 import LocalStorageService from 'services/LocalStorageService';
 
 export interface State {
   isAuth: boolean;
   language: string;
+  books: Page[];
```

Comment:
Why are you saving the books inside the UserContext? First you must verify that you actually need a context to save the books response and later if necessary then you must create a new context for this